osmnx/utils.py

```diff
@@ -62,7 +62,9 @@ def config(data_folder=settings.data_folder,
            default_crs=settings.default_crs,
            default_user_agent=settings.default_user_agent,
            default_referer=settings.default_referer,
-           default_accept_language=settings.default_accept_language):
+           default_accept_language=settings.default_accept_language,
+           nominatim_endpoint=settings.nominatim_endpoint,
```

Comment:
Can you add these new function arguments to the docstring "parameters" section below?

Reply:
@gboeing  I see you already did the change. Thank you! I will make sure I include docstring changes next time.